conf/ssl.conf

```diff
@@ -1,14 +1,17 @@
 # $Id: ssl.conf -1   $
 
-SSLEngine on
-SSLCertificateChainFile /etc/pki/tls/certs/StartSSLCAClass2.pem
-SSLCertificateFile /etc/pki/tls/certs/data.fm.cer
-SSLCertificateKeyFile /etc/pki/tls/private/data.fm.key
-#SSLOptions +StdEnvVars +ExportCertData
-SSLVerifyClient optional_no_ca
+<IfModule mod_ssl.c>
```

Comment:
It presents that using the `Let's Encrypt` service to generate the SSL certificate files and the wildcard certificate files.